inc/classes/admin/settings/class-beacon.php

```diff
@@ -327,12 +327,8 @@ public function get_suggest( $doc_id ) {
 			],
 			'defer_js'               => [
 				'en' => [
-					'id'  => '5d52138d2c7d3a68825e8faa',
-					'url' => 'https://docs.wp-rocket.me/article/1265-load-javascript-deferred/?utm_source=wp_plugin&utm_medium=wp_rocket',
-				],
-				'fr' => [
-					'id'  => '5d5ac08b2c7d3a7920be3649',
-					'url' => 'https://fr.​docs.​wp-rocket.​me/article/1270-chargement-differe-des-fichiers-js/?utm_source=wp_plugin&utm_medium=wp_rocket',
+					'id'  => '5e1d27de2c7d3a7e9ae627e8',
+					'url' => 'https://docs.wp-rocket.me/article/1304-remove-jquery-migrate/?utm_source=wp_plugin&utm_medium=wp_rocket',
```

Comment:
You need to keep the data for Defer JS, and add a new one for remove jQuery migrate. Then reference this new key in the `class-page.php` where the option is added

Reply:
Thank you!